lxd/device/disk.go

```diff
@@ -252,6 +252,8 @@ func (d *disk) validateConfig(instConf instance.ConfigReader) error {
 				if d.config["path"] != "" {
 					return fmt.Errorf("Custom block volumes cannot have a path defined")
 				}
+			} else if d.config["path"] == "" {
+				return fmt.Errorf(`Custom filesystem volumes require the "path" property`)
```

Comment:
thanks @monstermunchkin 

Please can you modify either the new error or the one above it so they are consistent, right now the old one says "cannot have a path defined" (no quotes on path, doesn't mention "property") and the new one quotes "path" and calls it a property.

Neither are wrong, but would be good to have them consistent.

Reply:
Fixed

Reply:
cheers!